src/Microsoft.DotNet.Interactive.CSharpProject/Packaging/AnalyzerResultExtensions.cs

```diff
@@ -3,35 +3,35 @@
 
 using System;
 using System.IO;
-using Buildalyzer.Workspaces;
 using System.Linq;
 using System.Runtime.CompilerServices;
-using Buildalyzer;
 using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.DotNet.Interactive.CSharpProject.RoslynWorkspaceUtilities;
 using Microsoft.DotNet.Interactive.CSharpProject.Servers.Roslyn;
+using static Microsoft.DotNet.Interactive.CSharpProject.RoslynWorkspaceUtilities.RoslynWorkspaceUtilities;
 
 //adpated from https://github.com/daveaglick/Buildalyzer/blob/master/src/Buildalyzer.Workspaces/AnalyzerResultExtensions.cs
 
 namespace Microsoft.DotNet.Interactive.CSharpProject.Packaging;
 
 public static class AnalyzerResultExtensions

```

Comment:
Do all of these need to be public?

Reply:
If this is only used internally, then we can delete the complete file. There are zero references to it.

Reply:
removed